### PR TITLE
Fixbug UI

### PR DESCRIPTION
**Pull request by guyyoo**
fixed UI bug in src\App\components\FileSelector.tsx and src\App\components\BookSelector.tsx

### CODE REVIEW

File: src/App/components/BookSelector.tsx [hunk 1]
```diff
@@ -28,12 +28,21 @@ export default function BookSelector(props: any): JSX.Element | null {
...
+                  project.setActiveBook(book.name);
+                }}
                 active={project.activeBookName === book.name}
               >
                 {book.name}
                 {book.isSelected && project.bookSelection.length > 1 && (
-                  <Tag position="absolute" zIndex={2} right="-10px" top="-10px"round intent={Intent.SUCCESS}>
+                  <Tag
+                    position="absolute"
+                    style={{ zIndex: 2 }}
```
Comment (iamam34) [inline src/App/components/BookSelector.tsx:40]: Note: the only non-formatting change in this whole branch is the replacement of `zIndex={2}` with `style={{zIndex:2}}` right here.

Comment (iamam34) [reply src/App/components/BookSelector.tsx:40]: I'm concerned that there doesn't seem to be a corresponding change in FileSelector.tsx. Can somebody who is familiar with the issue please explain what to look for?

**Comment by iamam34** [comment]
I can't see any warnings about the zIndex. I suggest closing without merge.

**Comment by irahopkinson** [comment]
This issue can't be reproduced currently. Closing.

**Comment by JCFarrow** [comment]
This error can be reproduced by selecting multiple chapters across multiple books. The zIndex value ensures that the green circles don't underlap:
image

**Comment by megahirt** [comment]
Our code has changed enough that an issue should be created and a new attempt at the change be submitted.

**Pull request closed without merging**